lib/newrelic-praxis/praxis_blueprints/render_event.rb

```diff
@@ -4,7 +4,11 @@ module Blueprint
       class RenderEvent < Event
 
         def metric_name
-          view_name = self.payload[:view].name
+          view_name = if self.payload[:view]
```

Comment:
since this is called often, should we do

```ruby
view_name = if view = self.payload[:view]
     view.name
else
    'fields'
end
```

instead?
I guess it is nothing in the grand scheme of things...but...why not? :)